apps/yggdrasil/priv/repo/migrations/20160229163000_create_role_permission.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Yggdrasil.Repo.Migrations.CreateRoleResource do
+  use Ecto.Migration
+
+  def change do
+    create table(:roles_permissions, primary_key: false) do
+      add :role_id, references(:roles, on_delete: :nothing)
+      add :resource, references(:resources, on_delete: :nothing, column: :name, type: :string)
+      add :permission, references(:permissions, on_delete: :nothing, column: :name, type: :string)
```

Comment:
@copenhas so this would usually be called `resource_name` and `permission_name`, but since it carries the name of the permission in this field and is basically the permission/resource respectively I opted to bend the rules a little here. 

It reads better to me, but I think I can still call this `:resource_name` or `:permission_name` and on the model alias as `resource` and `permission`.

Thoughts, wouldn't normally do this but it feels like it better suits how this table will primarily be used, with the the resource and permission field probably being used directly instead of looking up the actual record in the respective tables.